lib/active_interaction/concerns/runnable.rb

```diff
@@ -101,6 +101,7 @@ def run!
 
       e = InvalidInteractionError.new(errors.full_messages.join(', '))
       e.interaction = self
+      e.set_backtrace(errors.backtrace) if errors.backtrace
```

Comment:
I have one question here: can we guarantee that the type of `errors` here is always the `ActiveInteraction::Errors`? 

- If so the line is good
- otherwise we may want to do offensive coding like below

```suggestion
      e.set_backtrace(errors.backtrace) if errors.respond_to?(:backtrace) && errors.backtrace
```

Reply:
It's expected to be an `ActiveModel::Errors` class or a child of that (which `ActiveInteraction::Errors` is). I think this is fine.

Reply:
Wait, I lied. Because it could be an `ActiveModel::Errors` class without `backtrace` we'll want this.